views/index.erb

```diff
@@ -2,12 +2,19 @@
 <!-- node status on /clusters page -->
 <% @clusters.each_with_index do |cluster, index | %>   
       <div class="<%= 'col-sm-offset-3' if index == @clusters.count - 1 && @clusters.count.odd? %> col-sm-6 col-xs-12">
+          <% if GANGLIA_CLUSTERS[cluster.cluster_id].nil? %>
+            <div class="panel panel-default bs-callout bs-callout-info text-center panel-unclickable">
+              <h4><strong><%= cluster.cluster_title %> Cluster Status</strong></h4>
+              <%= erb :_node_status, :locals => { :showqer => cluster } %>
+            </div>
```

Comment:
create a partial with a local "clickable" which is true or false - that's the only thing that changes between the two. Make sure whitespace is good also.